ui-tests/src/test/java/io/syndesis/qe/steps/other/ODataSteps.java

```diff
@@ -42,8 +42,10 @@ public void insertEntityToODataService(final String entity, final String collect
     }
 
     @Then("^.*check that \"([^\"]*)\" entity in \"([^\"]*)\" collection contains$")
-    public void oDataServiceContains(final String entityKey, final String collection, DataTable table) {
-        // Currently supports only entityKey in number format
+    public void oDataServiceContains(String entityKey, final String collection, DataTable table) {
+        if (entityKey.matches("[A-Za-z]*")) {
```

Comment:
What is the purpose of this?

Reply:
just very simple check if the entityKey contains characters, so it includes `'`entityKey`'`

Reply:
yeah, but why do we do that? If the entity key is an empty string or just letters of the alphabet, we quote it, and if it's something else, we don't?

Reply:
if its just characters, we quote it so the odata service interprets it as string. If it is a number, we dont quote it. The check could be done differently, but this seems to work ok.

Reply:
of course this is just a dumb check as strings containing both characters and numbers is not matching this. But for the purpose of our tests this works just fine.

Reply:
in fact it's me who's dumb, this is perfectly fine :D